src/Infrastructure/Persistence/Initialization/DatabaseInitializer.cs

```diff
@@ -72,13 +72,15 @@ private async Task SeedRootTenantAsync(CancellationToken cancellationToken)
                 MultitenancyConstants.Root.Id,
                 MultitenancyConstants.Root.Name,
                 _dbSettings.ConnectionString,
-                MultitenancyConstants.Root.EmailAddress);
+                MultitenancyConstants.Root.EmailAddress,
+                isRoot: true);
 
             rootTenant.SetValidity(DateTime.UtcNow.AddYears(1));
 
             _tenantDbContext.TenantInfo.Add(rootTenant);
 
             await _tenantDbContext.SaveChangesAsync(cancellationToken);
+            await InitializeApplicationDbForTenantAsync(rootTenant, cancellationToken);
```

Comment:
Why??? 

This happens right after this method in the InitializeDatabasesAsync call...

This method is called "SeedRootTenant"... so shouldn't do anything else than seeding the root tenant...

Reply:
if you add more tenant like tell you order of processing is not like order of inserting ... and root is must the first processed

Reply:
Why would the order matter?